supabase_py/lib/Storage/StorageBucketApi.py

```diff
@@ -0,0 +1,104 @@
+import requests
+from requests import HTTPError
+
+
+class StorageBucketApi:
+
+    def __init__(self, url, headers):
+        self.url = url
+        self.headers = headers
+
+    def list_buckets(self):
+        """Retrieves the details of all Storage buckets within an existing product."""
+        try:
+            response = requests.get(f"{self.url}/bucket", headers=self.headers)
+
+            # If the response was successful, no Exception will be raised
+            response.raise_for_status()
+        except HTTPError as http_err:
+            print(f'HTTP error occurred: {http_err}')  # Python 3.6
```

Comment:
/nit  standardize all quotes to " instead of '
```suggestion
            print(f"HTTP error occurred: {http_err}")  # Python 3.6
```